swarm/pss/pss_test.go

```diff
@@ -764,12 +774,16 @@ func testNetwork(t *testing.T) {
 		}
 	}
 
+	time.Sleep(1 * time.Second)
+
```

Comment:
Why are there three sleeps spread out. Specifically, why are there sleeps both before and after this declaration?

Reply:
the sleeps are not before/after the `triggerChecks`, but rather:
1. after simulation is bootstrapped.
2. after getting the clients for all nodes.
3. after setting up the workers.

2 is probably not 100% necessary, I have to experiment with it.

however it is definitely the case that the simulation network is not waiting to be ready before we fire the tests, since those sleeps are making tests more reliable. we should investigate further where this asynchronicity is coming from.